CONTRIBUTING.md

```diff
@@ -5,69 +5,58 @@
 with the contribution process, please feel free to contact developers [on the
 mailing list or IRC](README.md#community). Note that larger Triton project
 changes are typically designed and discussed via ["Requests for Discussion
-(RFDs)"](https://github.com/joyent/rfd).
+(RFDs)"](https://github.com/TritonDataCenter/rfd).
 
-Triton repositories use the [Joyent Engineering
-Guidelines](https://github.com/joyent/eng/blob/master/docs/index.md). Notably:
+Triton repositories use the [Triton Engineering
+Guidelines](https://github.com/TritonDataCenter/eng/blob/master/docs/index.md).
+Notably:
 
 * The #master branch should be first-customer-ship (FCS) quality at all times.
   Don't push anything until it's tested.
 * All repositories should be "make check" clean at all times.
 * All repositories should have tests that run cleanly at all times.
 
-"make check" checks both JavaScript style and lint.  Style is checked with
-[jsstyle](https://github.com/davepacheco/jsstyle).  The specific style rules are
-somewhat repo-specific.  See the jsstyle configuration file or `JSSTYLE_FLAGS`
-in Makefiles in each repo for exceptions to the default jsstyle rules.
-
-Lint is checked with
-[javascriptlint](https://github.com/davepacheco/javascriptlint). Repos sometimes
-have repo-specific lint rules, but this is less common -- look for
-"tools/jsl.node.conf" for per-repo exceptions to the default rules.
-
-
 ## Issues
 
 There are two separate issue trackers that are relevant for Triton code:
 
-- An internal-to-Joyent JIRA instance.
+* An internal JIRA instance.
 
-  A JIRA ticket has an ID like `IMGAPI-536`, where "IMGAPI" is the JIRA project
+  A JIRA ticket has an ID like `TRITON-206`, where "TRITON-206" is the JIRA project
   name -- in this case used by the
-  [sdc-imgapi](https://github.com/joyent/sdc-imgapi) and related repos. A
-  read-only view of most JIRA tickets is made available at
+  [sdc-vmapi](https://github.com/TritonDataCenter/sdc-vmapi) and related
+  repos. A read-only view of most JIRA tickets is made available at
   <https://smartos.org/bugview/> (e.g.
-  <https://smartos.org/bugview/IMGAPI-536>).
-
-- GitHub issues for the relevant repo, e.g.
-  <https://github.com/joyent/triton/issues>.
+  <https://smartos.org/bugview/TRITON-206>).
+* GitHub issues for the relevant repo, e.g.
+  <https://github.com/TritonDataCenter/triton/issues>.
 
 Before Triton was open sourced, Joyent engineering used a private JIRA instance.
-While Joyent continues to use JIRA internally, we also use GitHub issues for
+While we continue to use JIRA internally, we also use GitHub issues for
 tracking -- primarily to allow interaction with those without access to JIRA.
+
+## Code of Conduct
+
+All persons and/or organizations contributing to, or intercting with our
+repositories or communities are required to abide by the
+[illumos Code of Conduct][coc].
+
+[coc]: https://github.com/TritonDataCenter/illumos-joyent/blob/master/CODE_OF_CONDUCT.md
```

Comment:
Add the code-of-conduct bug to this summary as well prior to push.  (Just a line with the bug id on it!)